osu.Game/Input/KeyBindingStore.cs

```diff
@@ -60,6 +63,18 @@ private void insertDefaults(IEnumerable<KeyBinding> defaults, int? rulesetId = n
                         usage.Context.SaveChanges();
                     }
                 }
+
+                // for now this should be fine here
```

Comment:
what should be fine where?

comments should explain what code is doing, not make statements about something you're thinking at the time. an explanation of why this is here of what it's doing should precede your "should be fine" statement, at very least.

Reply:
Will expand this, sorry. I think that code doesn't belong in this method at all, but for the time being it fits there just fine. It would get removed anyways in a future release.